### PR TITLE
zipModule: check zip enviroment path existence

### PR DESCRIPTION
**Pull request by CeresBarros**
When zip paths are not defined in Sys.env `zip` doesn't work, but also doesn't throw any error - very annoying! Having a little check may save hours of trying to understand why zip is not working

### CODE REVIEW

File: R/module-template.R [hunk 1]
```diff
@@ -947,6 +947,11 @@ setMethod(
   "zipModule",
   signature = c(name = "character", path = "character", version = "character"),
   definition = function(name, path, version, data, ...) {
+    dots <- list(...)
+    if (is.null(dots$zip) &
+        all(Sys.getenv(c("R_ZIPCMD", "zip")) %in% ""))
+      stop("External zip command paths missing.\nAdd 'zip = <path>/zip.exe' specifying path to zip.exe")
```
Comment (achubaty) [inline R/module-template.R]: This is a Windows-specific message, so ensure it's only shown on Windows.

**Comment by eliotmcintire** [comment]
Is this PR still relevant? @CeresBarros ?

**Comment by CeresBarros** [comment]
Yes

**Comment by CeresBarros** [comment]
I guess the windows-specific message needs to be added still, though. In any case, this fix enabled me to re-zip modules again today (I forgot, again, that I need to supply the zip.exe path)

**Pull request merged**